packages/plugins/rubygem-infoblox/rubygem-infoblox.spec

```diff
@@ -1,76 +1,85 @@
-# Generated from infoblox-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name infoblox
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 3.0.0
-Release: 1%{?dist}
-Summary: A Ruby wrapper to the Infoblox WAPI
+Release: 2%{?dist}
+Summary: This gem is a Ruby interface to the Infoblox WAPI. Using the gem, you can query, create, update, and delete DNS records in your Infoblox instance
```

Comment:
I think summary and description should be swapped?

Reply:
I agree with you, but this is generated by the gem2rpm template and any future updates will keep putting this back the way it is.

Reply:
We don't regen the template that often. When bumping dependencies it only replaces the block between `# start specfile generated dependencies` and `# end specfile generated dependencies`. I've submitted https://github.com/Granicus/infoblox/pull/62 so it should be clean in future releases though.